app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileRenameActivityTest.kt

```diff
@@ -118,6 +126,21 @@ class ProfileRenameActivityTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  @Test
+  fun testProfileRenameActivity_hasCorrectActivityLabel() {
+    activityTestRule.launchActivity(
+      ProfileRenameActivity.createProfileRenameActivity(
+        context = context,
+        profileId = 1
+      )
+    )
+
+    val title = activityTestRule.activity.title
+    assertThat(title).isEqualTo(
+      context.getString(R.string.profile_rename_activity_label)
+    )
```

Comment:
Try to make it fit in a single line, if it is exceeding the 100 char limit, then it is fine.

Reply:
DONE